<launch>
    <arg name="rviz" default="true" />
    <env name="ROSCONSOLE_CONFIG_FILE" value="$(find voxel_mapping)/conf/run.conf" />
    <node pkg="voxel_mapping" type="voxel_mapping_node" name="voxel_mapping" output="screen">
        <param name="map_frame" value="map" />
        <param name="scan_topic" value="/laser/scan" />
        <param name="scan_frame" value="laser" />
        <param name="resolution" value="0.05" />
        <param name="plane_threshold" value="0.1" />
        <param name="max_iteration" value="10" />
        <param name="update_thresh" value="10.0" />
        <param name="max_update_thresh" value="1000.0" />
        <param name="ignore_range" value="0.35" />
        <param name="max_voxel_number" value="100000" />
    </node>
    <group if="$(arg rviz)">
        <node pkg="rviz" type="rviz" name="rviz" args="-d $(find voxel_mapping)/rviz/default.rviz" />
    </group>
</launch>